routes/config.route.js

```diff
@@ -38,7 +38,18 @@ routes.route('/').post(async function (req, res) {
 
     res.status(204);
     res.send();
-    console.log('Config save successful');
+    console.log('Config saved successfully');
+});
+
+routes.route('/').delete(async function (req, res) {
+    const deleteQuery = ConfigModel.deleteMany();
+    await deleteQuery.exec(function (err) {
+        if (err) { throw err; }
+    });
+
+    res.status(204);
+    res.send();
+    console.log('Config deleted successfully');
```

Comment:
Why you use `console.log`, it's for logging ?
The http code for rest endpoint is enough to inform the user (the documentation can help if necessary)